Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2444,7 +2461,13 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
     @inlinable
     @differentiable(wrt: self)
     func standardDeviation(squeezingAxes axes: Tensor<Int32>) -> Tensor {
-        TensorFlow.sqrt(variance(squeezingAxes: axes))
+        precondition(
```

Comment:
The entire `precondition` can fit on one line:
```suggestion
        precondition(areAxesInRange(axes), "All axes must be in the range `[-rank, rank)`.")
```

Please update other `precondition`s similarly.